scripts/apps/search/components/Item.jsx

```diff
@@ -114,11 +114,20 @@ export class Item extends React.Component {
             'sd-grid-item sd-grid-item--with-click' :
             'media-box media-' + item.type;
 
+        let multiSelected = this.props.item.selected;
```

Comment:
imo would be better to avoid this variable altogether, because it's not what it says and the way it's used is kind of confusing. so for `div.selected` there would be:
`selected: this.props.item.selected || this.props.flags.selected,`
and for `li.selected`:
`selected: this.props.item.selected && !this.props.flags.selected,`